### ISSUE TITLE
Missing status some icons

### ISSUE THREAD

**Issue by FadeMind** [varlesh/papirus-pack-kde#30]
Hi You made great work. Thanks for papirus dark ;)

In KDE Plasma 5 some icons are missing in Papirus theme (dark and non-dark theme OFC)

in **status** directory:

- ![folder-open](http://i.imgur.com/iX9bFGS.png)
folder-open
-  ![wallet-co](http://i.imgur.com/oTEI4Tu.png)
wallet-closed
wallet-open
- ![mail](http://i.imgur.com/VxczBg7.png)
mail-sent
mail-tagget
mail-task
mail-attachment
mail-queued
mail-read
mail-replied
mail-unread-new
mail-unread
- ![battery](http://i.imgur.com/jn3KIxs.png)
battery-040
battery-060
battery-080
battery-100
battery-caution
battery-charging-040
battery-charging-060
battery-charging-080
battery-charging-caution
battery-charging-low
battery-charging
battery-missing
battery-low
- ![meet](http://i.imgur.com/X1nwzq6.png)
meeting-participant
meeting-chair
meeting-observer
meeting-participant-optional
meeting-participant-reply
meeting-organizer
![wt](http://i.imgur.com/RcrOx7J.png)
weather-hail
weather-many-clouds
weather-clear-night
weather-clear
weather-clouds-night
weather-clouds
weather-few-clouds-night
weather-few-clouds
weather-mist
weather-showers
weather-showers-day
weather-showers-night
weather-showers-scattered
weather-showers-scattered-day
weather-showers-scattered-night
weather-snow-rain
weather-freezing-rain
weather-snow-scattered
weather-snow-scattered-day
weather-snow-scattered-night
weather-none-available
weather-snow
weather-storm-day
weather-storm-night
weather-storm
- ![task](http://i.imgur.com/bDk3rI9.png)
mail-task
task-delegate
task-complete
task-ongoing
task-reject
task-reminder
task-accepted
task-recurring
- ![appoinment](http://i.imgur.com/jk5t6WP.png)
appoinment-recurring
appoinment-reminder
- ![print](http://i.imgur.com/cWdM3OM.png)
printer-printing
printer-error
- ![lock](http://i.imgur.com/Oe95TN1.png)
object-locked
object-unlocked
- ![il](http://i.imgur.com/5tixcPf.png)
image-loading
- ![md](http://i.imgur.com/DJjfWz2.png)
media-playlist-repeat
media-playlist-shuffle

Regards

**Comment by varlesh** [comment]
Why many icons not working from you?
This icons already added:
folder-open
wallet-open
mail-sent
mail-task
mail-attachment
mail-read
mail-replied
mail-unread-new
mail-unread
object-locked
object-unlocked
media-playlist-repeat 
media-playlist-shuffle

Where realy used battery* icons?
All battery icons available on plasma theme.
https://github.com/varlesh/papirus-pack-kde/blob/master/plasma-themes/papirus/icons/battery.svgz

**Comment by varlesh** [comment]
You use kde DE?

**Comment by FadeMind** [comment]
Just click properties of DIRECTORY and change icons of it. Here it is all.
battery icons are not honored at all (just missing)
See:
![img](http://i.imgur.com/0Mn7XhV.png)

I running Plasma 5 KDE on Arch.

See my customized zephyrflattr icon set. This have fixed this issues (mostly): https://github.com/FadeMind/FaenzaFlattr2-Zephyr/tree/master/status/scalable

And one more: lots of GNOME apps need to be merged icons names... 

Compare Your icon set with this listing:
https://github.com/FadeMind/FaenzaFlattr2-Zephyr/blob/master/FILES

Regards

Shutter app using custom icons from GNOME stock if will not find:
![shutt](http://i.imgur.com/68itzDC.png)
In Help menu there are in status subdir::
`lpi-bug.svg`
`lpi-help.svg`
` lpi-translate.svg`

**Comment by varlesh** [comment]
Ok. Thx for contributing ) But
Some icons already existing on theme and issue too confusing.
Please clear your request and delete existing icons.

**Comment by FadeMind** [comment]
You are using different tree than *faenza-like*.
I will send here new issue request when I'll find out what directly is missing, for now please follow first post and add  missing battery icons (and other symlinks) .

Co-operation begins ;)

PS.:

Look in changing icon of any dir and look around:
![a](http://i.imgur.com/9932MCq.png)

**Comment by varlesh** [comment]
Your request very big and confusing. Many icons already exist. 
Please organize your issue. 
For example:

Acions
media-playlist-repeat
media-playlist-shuffle

Status:
weather-hail
weather-many-clouds
etc

**Comment by FadeMind** [comment]
Missing icons in **status** directory
folder-open
wallet-closed
wallet-open
mail-sent
mail-tagget
mail-task
mail-attachment
mail-queued
mail-read
mail-replied
mail-unread-new
mail-unread
battery-040
battery-060
battery-080
battery-100
battery-caution
battery-charging-040
battery-charging-060
battery-charging-080
battery-charging-caution
battery-charging-low
battery-charging
battery-missing
battery-low
meeting-participant
meeting-chair
meeting-observer
meeting-participant-optional
meeting-participant-reply
meeting-organizer
weather-hail
weather-many-clouds
weather-clear-night
weather-clear
weather-clouds-night
weather-clouds
weather-few-clouds-night
weather-few-clouds
weather-mist
weather-showers
weather-showers-day
weather-showers-night
weather-showers-scattered
weather-showers-scattered-day
weather-showers-scattered-night
weather-snow-rain
weather-freezing-rain
weather-snow-scattered
weather-snow-scattered-day
weather-snow-scattered-night
weather-none-available
weather-snow
weather-storm-day
weather-storm-night
weather-storm
mail-task
task-delegate
task-complete
task-ongoing
task-reject
task-reminder
task-accepted
task-recurring
appoinment-recurring
appoinment-reminder
printer-printing
printer-error
object-locked
object-unlocked
image-loading
media-playlist-repeat
media-playlist-shuffle

What is unclear?

**Comment by FadeMind** [comment]
DIFF files are just too big for ONE commit issue. No worries. 
DIFFS tree between papirus (---) and faenzaflattr (+++)
MIMETYPES: https://pastebin.com/diZjGy6u
STATUS: https://pastebin.com/psWAKVu7
PLACES: https://pastebin.com/T5qM81vf
ACTIONS: https://pastebin.com/ns69uvXr

Regards

**Comment by FadeMind** [comment]
Battery icons are used from oxygen theme (if are missed in used theme), not from https://github.com/varlesh/papirus-pack-kde/blob/master/plasma-themes/papirus/icons/battery.svgz SVGZ file.
SVGZ file is used by Plasma Shell theme in tray. Nowhere more.

See dirs content:
/usr/share/icons/oxygen/64x64/status/
/usr/share/icons/oxygen/48x48/status/

**Comment by varlesh** [comment]
Why you post AGAIN already exist icons and not check this?
folder-open already exist in places
https://github.com/varlesh/papirus-pack-kde/blob/master/icons/papirus/48x48/places/folder-open.svg
mail-sent already exist as symlink
https://github.com/varlesh/papirus-pack-kde/blob/master/icons/papirus/16x16/actions/mail-sent.svg
media-playlist-repeat already exist on actions
https://github.com/varlesh/papirus-pack-kde/blob/master/icons/papirus/16x16/actions/media-playlist-repeat.svg
media-playlist-shuffle already exist on actions
https://github.com/varlesh/papirus-pack-kde/blob/master/icons/papirus/16x16/actions/media-playlist-shuffle.svg
etc etc...
PAPIRUS icon theme based on PAPER icon theme. I can't create icons based on Oxygen and other themes standard.

**Comment by varlesh** [comment]
I understand that you want a full set icons . But my time and abilities limited. I can't have many developers and desingers. I work alone from this :(

**Comment by varlesh** [comment]
Why you not use Numix Circle as standard? Because Numix Circle have many developers and mostly popularity... But Numix Circle have Full pack icons during all this time.
Please have patience and check you issue ))

**Comment by FadeMind** [comment]
I can help You figure out. PAPER have these same issue - missing lots of XDG standard icon (not only oxygen). 
Lots of time taken fixed missing icons in Breeze... but OFC KDE only target. 
Papirus is great. I want help you figure out. 

I helped previous developer of FaenzaFlattr (tago). Now this project is ended (but I forked them and fixed alone). It is based on Faenza SVG dir structure, without custom subdirs). 

Lots of mimes I figure out alone andd just sent commit to tago. 
Lots of customized icons set have lacks.  
Papirus like I said is good job. Clean and good looking. Good Job. 

Numix have this missing too... but I must check current status  of it.

tl:dr If You don't want fixing this OK. Nothing urgent. 
If You need contributors, ask of them on reddit/facebook etc.

Papirus is good. But can be better if You'll ask for contrib.

**Comment by varlesh** [comment]
if use freedesktop standard I have to reorganize all icons and folders over again. This very hard way and very laborious.
And my pack not fully:
- Not support HDPI
- Places icons have only 48 and 16px (need 22, 24, 32 etc)
- Apps icons have only 48 and 32px (need other sizes)
- Many actions icons not available and port to  22, 24, 32 and other sizes.
There are a lot of work.

**Comment by varlesh** [comment]
For example. I have 385 actions icons with 16x16px.
16x16 can crop to 32x32, 64x64, 128x128 and 256x256
But i need create new icons for 22x22 and 22x24 (which can then be crop to 48x48 and 96x96)
I need create new 385 icons!!! This only actions, but there are other - apps, places, status etc...
Literally thousands icons need create and ported.

**Comment by varlesh** [comment]
It's impossible for one man.
No developers - not standard, not full icon set, no HDPI and etc.

**Comment by FadeMind** [comment]
In next weekend I'll take care of this... be patient... 
Better is using scalable SVG and customize them directly in index.theme files, than create each one of them separately.. 

ONE MORE thing.... I'll improve Your PKGBUILD in future. Downloading whole repo everytime for separate package is costs for 3G...

EVERYTHING is possible... REALLY.

Take me little time to setup plan, next I'll mailing to You. I closing this issue for now. 

Cheers

**Comment by horosgrisa** [comment]
@FadeMind 
Is it possible to download only a part of the repository?

**Comment by varlesh** [comment]
Scalable icons blured and not like me. But no other way

**Comment by FadeMind** [comment]
@horosgrisa  @varlesh need to create a separate repo for each one project (plasma 5 theme, look and feel, yakuake, konsole, libreoffice, icon set) or just create merged PKGBUILD whitch one will be just download ONE time all stuffs and will be create all separated packages in ONE time.

See https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=libappindicator-activate

There is ONE packagebase: libappindicator-activate whitch You create 3 separated packages: 'libappindicator-activate-gtk2' 'libappindicator-activate-gtk3' 'libappindicator-activate-sharp'

For papirus it is possible handle out the same way.

**Comment by varlesh** [comment]
@horosgrisa я совсем недавно только создал этот репозиторий. Опять переносить... что-то не хочется

**Comment by horosgrisa** [comment]
@FadeMind OK, I will merge some packages.
@varlesh  И мне тоже так кажется

**Comment by varlesh** [comment]
@horosgrisa Может с лаунчпада брать (для kubuntu который) или kde-look к примеру?

**Comment by horosgrisa** [comment]
@varlesh  Нет, я просто объединю  несколько пакетов в один

**Comment by varlesh** [comment]
Хотя kde-look плохая идея, потому как там лимит на объем файл в 5mb, а уже в tar.xz 4.8 при последнем заливе. Сжать сильней не получается...

**Comment by horosgrisa** [comment]
@varlesh  Хотя я и не так часто обновляю PKGBUILD

**Comment by varlesh** [comment]
@horosgrisa а тут есть личка или что-то подобное вы не в курсе?

**Comment by horosgrisa** [comment]
@varlesh Вроде нет

**Comment by varlesh** [comment]
@horosgrisa  Жаль. Если будет не трудно и есть время, то подтвердите этот запрос
https://github.com/varlesh/papirus-pack-kde/issues/29
Не пойму почему она черная, все перепроверил уже несколько раз.

**Comment by horosgrisa** [comment]
@varlesh Steam 32-битный, а я не хочу засорять систему 32-битными библиотеками, поэтому не могу помочь.

**Comment by horosgrisa** [comment]
@FadeMind You can use my repository
```
[udom]
SigLevel = Never
Server = http://repo.udom.tk
```
This repo support deltas.

**Comment by FadeMind** [comment]
@varlesh  @horosgrisa 

https://github.com/FadeMind/archpkgbuilds/blob/master/aur4-customized/papirus/PKGBUILD

PKGBUILD for LOCAL build all papirus package based on lastest GIT snapshots.

Available on AUR too.
